hpmor.tex

```diff
@@ -145,6 +145,7 @@ \part{Harry James Potter-Evans-Verres and the Philosopher's Stone}
 \include{chapters/hpmor-chapter-111}
 \include{chapters/hpmor-chapter-112}
 \include{chapters/hpmor-chapter-113}
+\include{chapters/hp-exam}
```

Comment:
I prefer leaving the exam at the end rather than inserting it into the story, as it's not part of the story; it feels like an appendix.

Reply:
This is a matter of taste, I very much liked to see it directly in place, since this gave me some idea of what has happened at the time of posting around the globe and reddit community ;-)

Reply:
I agree it's a matter of taste! For me, having it "in place" makes sense in the original online version of the book: it's a transcript of the original process of writing (and perhaps you can also see all the original comments from readers as it was written?).

One could certainly imagine a "book edition" that attempted to recreate this.

For me, *this* "book edition" of HPMOR is, instead, an attempt at a "finished product", that is, the book resulting from the authoring process: a complete, self-contained story that one can read in its own world. Having the extra material (such as the final exam and the omake) is interesting, and may direct readers to the wider community, but it's not a central part of *this* experience.

It might be worth thinking about having multiple "configurations" of the book. It is certainly possible to do with LaTeX, after all! But that would make the project more complicated, and it would be necessary to think carefully about how to explain this to readers.

Reply:
oki, reverted for now ;-)